database/files/20211119_233423_agregarcampoDetalle_movimiento_detalle_down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE movimientos.movimiento_detalle DROP COLUMN IF EXISTS detalle;
```

Comment:
(En general) Dejar la última línea de cualquier archivo de texto plano seguido por Git, vacía, sin espacios ni nada
```suggestion
ALTER TABLE movimientos.movimiento_detalle DROP COLUMN IF EXISTS detalle;

```